config.php

```diff
@@ -190,12 +202,19 @@ public function _search_users($search, $exact=false) {
             }
             $filter .= "(|$subfilter)";
         }
+        if (LDAP_EXCLUDE != '') {
+            $filter = $filter . LDAP_EXCLUDE;
+        }
         $filter = "(&$filter)";
       }
     } else {
       $escaped = escape_ldap_filter_value($search);
       $filter = "(mail=$escaped)";
     }
+    if(!$this->phonebook_admin){
```

Comment:
I was reusing existing code from elsewhere, but I'm happy to alter it since you asked. Note that we are not consistent with regards to this stylistic request:

```
$ git grep 'if(' | wc -l
     157
$ git grep 'if (' | wc -l
     638
```